src/Tickets/Commerce/Reports/Data/Order_Summary.php

```diff
@@ -21,7 +21,7 @@ class Order_Summary {
 	 *
 	 * @var int The post ID.
 	 */
-	protected int $post_id;
+	public int $post_id;
```

Comment:
Agh, no. 
If you need to have this `public` to read its value, add a getter method and leave this `protected`.
Whenever you make a property `public`, you should add all the code that, on each method is making sure that property value, which can then be changed from any piece of code at any moment, still makes sense.